packages/kbn-dev-utils/src/plugins/parse_kibana_platform_plugin.ts

```diff
@@ -64,6 +63,12 @@ export function parseKibanaPlatformPlugin(manifestPath: string): KibanaPlatformP
     throw new TypeError('expected new platform plugin manifest to have a string version');
   }
 
+  if (!manifest.owner || typeof manifest.owner.name !== 'string') {
```

Comment:
Our doc says 
> Every *internal* plugin should fill in the owner and description properties.`

https://github.com/elastic/kibana/blob/9ecdbeb58c4ba521ffc2beb7ef9e34d36008593e/dev_docs/tutorials/building_a_plugin.mdx#kibanajson

But here `owner` is required for 3rd party plugins as well. Should we either apply this check to the Elastic plugins only or update the doc and explain what information should provide this field for 3rd party plugins?
For example, what should be `name` in this string? a GitHub account name? What if a person doesn't have one? 
For comparison, `package.json` field `author` provides several different fields: `name`, `email` ,`url`. https://docs.npmjs.com/cli/v7/configuring-npm/package-json#people-fields-author-contributors

Reply:
Ah. I don't think we have a way to distinguish between internal and external plugins (@spalger ?), so I'd be fine with just making this required and updating that doc. They can put whatever they like, their name, email, or GitHub account name. Since it's only used for our API documentation, it won't make a difference.

We considered `author` but I felt `owner` is a better fit because a plugin can be developed by one team and passed off to another team to maintain.

Reply:
No idea if the core keeps track of what is internal vs external but I know that someone was working on new validation around dependencies from src/plugins to x-pack/plugins, and if I recall correctly someone said that they plan it to track where the plugins came from... cc @pgayvallet